.editorconfig

```diff
@@ -0,0 +1,21 @@
+root = true
+
+[*]
+end_of_line = crlf
```

Comment:
This shouldn't be necessary with new-line normalization in `.gitattributes`. Also new-line terminators vary with platforms.